tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -3794,29 +3590,31 @@ Status FragmentMetadata::load_footer(
   if (loaded_metadata_.footer_)
     return Status::Ok();
 
-  Buffer buff;
-  shared_ptr<ConstBuffer> cbuff = nullptr;
-  if (f_buff == nullptr) {
+  Tile tile;
```

Comment:
This will call the default constructor for Tile, which we should avoid as I'm going to remove the default constructor for tile in a PR soon... That's why I suggested a call to get_footer_offset_and_size when f_tile == nullptr so you can create a tile with the right size right away.